Update syntax and remove ad limitation for a few tests.

recuero opened the pull request:
Refs. #20090

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1072434/) on a060595 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/21253/site)

This comment will be updated on new commits.

recuero committed:
Update syntax and remove ad limitation for a few tests. (idaholab#20090)

Removing the derivative entries from interior nodes has enabled running a couple
of tests with MOOSE default AD size.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1075449/) on ff59f59 wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>0973fa</th>
      <th colspan=4>#21253 ff59f5</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>83.42%</td>
      <td>83.41%</td>
      <td>-0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>77359</td>
      <td>77358</td>
      <td>-1</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>15380</td>
      <td>15381</td>
      <td>+1</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/21253/coverage_diff/framework)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/21253/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/contact)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/navier_stokes)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/richards)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/21253/coverage/xfem)

</details>



This comment will be updated on new commits.
